### PR TITLE
 Add module build artifact to the story

### PR DESCRIPTION
**Pull request by yashvardhannanavati**

### CODE REVIEW

**Comment by yashvardhannanavati** [comment]
modules

**Comment by ralphbean** [comment]
Rad!  @asamalik, have a look?

File: src/app/story/siblings/siblings.component.ts [hunk 1]
```diff
@@ -57,6 +57,9 @@ export class SiblingsComponent implements OnDestroy {
 
       this.selectedResource = this.route.snapshot.params['resource'];
       this.selectedUid = this.route.snapshot.params['uid'];
+      
+      let story_type: string;
+      story_type = this.route.snapshot.queryParams['story_type']
```
Comment (mprahl) [inline src/app/story/siblings/siblings.component.ts]: This variable isn't reassigned, so it's best to just do:
```typescript
const story_type = this.route.snapshot.queryParams['story_type']
```

or

```typescript
const story_type: string = this.route.snapshot.queryParams['story_type']
```

**Comment by mprahl** [comment]
@yashvardhannanavati looks great so far. Let me know if you need help on the tests.

**Comment by yashvardhannanavati** [comment]
@mprahl and @sarah256 could you please review this?

**Comment by yashvardhannanavati** [comment]
Thanks guys! :)

**Comment by mprahl** [comment]
:+1:

**Comment by sarah256** [comment]
:dancer: :+1:

**Pull request merged**